### DISCUSSION TITLE
One of my collection is gone

### DISCUSSION THREAD

**Discussion by MuslemRahimi** [pocketbase/pocketbase#3876] [Q&A] [answered]
Hi,

I hope you guys can help me out.
One of my collections is gone in pocketbase called 'notifications'. I cannot see it anymore in the UI. 

I use marmot for scaling but the author told me that marmot never drops or adds table to the databases.
I cleaned all traces from marmot as mentioned in my last discussion https://github.com/pocketbase/pocketbase/discussions/3867.

When i read from the collection via ```await pb.collection('notifications').getFullList()```I get an error 404.

Furthermore when I create a new collection with the same name I get the error
```
2023/12/04 11:48:08 SQL logic error: table `notifications` already exists (1)
```

**Comment by ganigeorgiev** [comment, marked as answer]
It looks like something has gone wrong with the syncing between your instances.

If you are not seeing the collection in the UI, then this means that the row in the `_collections` table describing the `notifications` records table is missing.

What you can do to fix it is to try to find a migration file that creates the `notifications` collection, duplicate it with a new timestamp prefix and replace the `SaveCollection()` call with just `Save()` so that you can save only the collection model without trying to update the related records table.

**Reply by MuslemRahimi** [reply]
Nice this works for me :+1:
